subsys/bluetooth/controller/Kconfig

```diff
@@ -143,6 +143,18 @@ config BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
 	  microseconds. The event length and the connection interval are the
 	  primary parameters for setting the throughput of a connection.
 
+config BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT
+	int "Default central ACL event spacing [us]"
+	default BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
+	depends on BT_CENTRAL
+	help
+	  On the central, sets the time ACL connections are spaced apart assuming that
+	  they are using the same connection interval.
+	  BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT may be set to a different value
```

Comment:
it's a bit confusing still.
Maybe something like BT_CTRL_... allows the application to set a different value for event spacing than ....